subsys/mgmt/mcumgr/smp_bt.c

```diff
@@ -41,6 +42,11 @@ static struct bt_uuid_128 smp_bt_svc_uuid = BT_UUID_INIT_128(
 static struct bt_uuid_128 smp_bt_chr_uuid = BT_UUID_INIT_128(
 	BT_UUID_128_ENCODE(0xda2e7828, 0xfbce, 0x4e01, 0xae9e, 0x261174997c48));
 
+/* It there is re-assembly in progress, this should be true. */
+#ifdef CONFIG_MCUMGR_SMP_PKASM_BT
+static bool in_progress;
```

Comment:
This is already tracked by the PKASM module. You can define a new PKASM API that will tell this layer whether reassembly is ongoing or not.

Reply:
bump